scdl/scdl.py

```diff
@@ -459,63 +457,118 @@ def download_track(track, playlist_name=None, playlist_file=None):
         fileToKeep.append(filename)
 
     # Download
```

Comment:
You can safely ignore all this garbage, it is simply reducing the indentation by 1 tab (4 spaces) from the line `if not os.path.isfile(filename):`